src/ui/utils/mixpanel.ts

```diff
@@ -147,6 +147,11 @@ export async function trackMixpanelEvent(...[event, properties]: [...MixpanelEve
     mixpanel.track(event, { ...properties, namespace: namespaceFromEventName(event) });
   }
 }
+export async function timeMixpanelEvent(event: string) {
```

Comment:
Only the event is relevant here. @abstractalgo got some TS-fu that can get us the right type?

Reply:
If I understand correctly, `event: MixpanelEvent[0]`?

Reply:
yep!